tools/maven-plugin/README.adoc

```diff
@@ -49,6 +49,7 @@ E.g. for includeDependenciesScopes could be configured as:
 - `configProperties` (String) - Load any properties from a file. Example `${basedir}/src/main/resources/application.properties`.
 - `attachArtifacts` (boolean, default: false) - Attach the built OpenAPI schema as build artifact.
 - `skip` (boolean, default: false) - Skip execution of the plugin.
+- `encoding` (String) - Encoding of output OpenAPI files.
```

Comment:
We can probably make the default UTF-8 to stay backward compatible ?

Reply:
@phillip-kruger, I thought the same thing, but previously it was using `String#getBytes`, which internally calls `Charset.defaultCharset`.

Reply:
Ok, but it should still be able to do this. Not sure if you can make these properties Optional ? If so , use Optional with empty setting to default

Reply:
The default `defaultValue` in the `@Parameter` is an empty string. That flows into the `(!StringUtils.isBlank(encoding))` check and then uses the platform default encoding.

Reply:
Ok great !

Reply:
So then we can merge ?

Reply:
Yeah, lgtm!